topology.go

```diff
@@ -15,7 +15,11 @@ var (
 )
 
 const (
-	generationsTableNamePre4_4 = "system_distributed.cdc_streams_descriptions"
+	systemDistributedKeyspace  = "system_distributed"
```

Comment:
I think using a fully qualified name is more elegant - what if in the future there is a new table in a different keyspace? The fact that all of these tables lie in system_distributed is kind of accidental.
But that's up to you.

Reply:
> I think using a fully qualified name is more elegant - what if in the future there is a new table in a different keyspace? The fact that all of these tables lie in system_distributed is kind of accidental. And now you have to do these string concatenations in the code, obscuring the logic.

Well, it's either string concatenation or string splitting - I need to use the unqualified name in order to check if the table is present in the schema - see `chooseGenerationSource` or `(*generationSourcePre4_4).maybeUpgrade`.

All in all, I think I'll change it so that names are fully qualified - there are more places in which concatenations are needed now than places which will need splitting, and you have a good point about possibility of future tables being in other keyspaces.

Reply:
Done, switched to fully qualified names